src/index.js

```diff
@@ -49,20 +30,13 @@ function articleDiv(article, articleTemplateSelector, options) {
 
   var articleImage;
 
-  if (
-    article._embedded &&
-    article._embedded["wp:featuredmedia"] &&
-    article._embedded["wp:featuredmedia"][0] &&
-    article._embedded["wp:featuredmedia"][0].media_details &&
-    article._embedded["wp:featuredmedia"][0].media_details.width &&
-    article._embedded["wp:featuredmedia"][0].media_details.height
-  ) {
+  if (article._embedded && article._embedded["wp:featuredmedia"] && article._embedded["wp:featuredmedia"][0] && article._embedded["wp:featuredmedia"][0].media_details && article._embedded["wp:featuredmedia"][0].media_details.width && article._embedded["wp:featuredmedia"][0].media_details.height) {
```

Comment:
I wonder can't we use an optional chaining as in:
```
article._embedded?.["wp:featuredmedia"]?.[0]?.media_details?.width && article._embedded?.["wp:featuredmedia"]?.[0]?.media_details?.height
```

Reply:
There is a setting in the build that is enforcing ES5, so we wont be able to do this. Same for the comments below